public/scripts/app.js

```diff
@@ -1,78 +1,69 @@
 $(document).ready(function() {
-
```

Comment:
CODE CLEANLINESS IS IMPORTANT.

This file has unnecessary comments (try to name functions more intuitively instead of adding comments). 

It also has a lot of unnecessary newlines and `console.log`s. 

Indentation is inconsistent.